src/index.ts

```diff
@@ -82,3 +82,5 @@ export { ApplicationFrame, Page, Sidebar } from "./Layout";
 export { useWindowDimensions } from "./utils";
 export { Divider } from "./Divider";
 export { SortingTable } from "./SortingTable";
+export { handleStyledProps } from "./StyledProps";
```

Comment:
Is the reason we export this is to make it possible for someone to create their own styled-component with `styled-system` attributes outside of NDS? 

If so, this might not work if the project does not include `styled-system` as a dependency no?

Reply:
Currently the only peer dependencies we require are `react`, `react-dom` and `styled-components`.

Reply:
I wanted to make new styled components in Eco that extended NDS types. In order for those to work with the styled-system props (and be able to override css properties set via styled-components) I had to define my own `handleStyledProps` function in Eco (which was not as good because it didn't have the NDS extensions). So I was thinking why not have NDS export this as a utility?

But you're right, I forgot to consider the peer dependency thing. I suspect that it will be fine even if the project does not include `styled-system` because I am currently able to use `styled-system` props on NDS components in Eco and Eco didn't have a direct dependency on `styled-system` until recently.

Reply:
I will update the documentation to list `styled-system` as an optional peer dependency if someone is to use these utility functions.